Podcast/SearchITunesViewController.swift

```diff
@@ -0,0 +1,134 @@
+//
+//  SearchITunesViewController.swift
+//  Podcast
+//
+//  Created by Mindy Lou on 11/1/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class SearchITunesViewController: ViewController, UITableViewDelegate, UITableViewDataSource, UISearchControllerDelegate, SearchSeriesTableViewDelegate, UISearchBarDelegate {
+    
+    var searchController: UISearchController!
+    var tableView: EmptyStateTableView!
+    
+    var searchResults: [Series] = []
+    let seriesCellIdentifier = "SeriesCell"
+    let seriesCellHeight: CGFloat = 95
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .paleGrey
+        title = "Search iTunes"
+        
+        tableView = EmptyStateTableView(frame: view.frame, type: .search)
+        tableView.dataSource = self
+        tableView.delegate = self
+        tableView.stopLoadingAnimation()
+        tableView.backgroundView?.isHidden = true
```

Comment:
isn't background view hidden the default for emptystatetableview

Reply:
yeah it is, but for some reason isHidden is false for this tableView